HEC.FDA.Model/stageDamage/ScenarioStageDamage.cs

```diff
@@ -2,6 +2,7 @@
 using Statistics;
 using HEC.FDA.Model.paireddata;
 using HEC.FDA.Model.interfaces;
+using Microsoft.VisualBasic;
```

Comment:
Why is the reference to VisualBasic being added? is it because of the way you are formatting this string?

Reply:
it was the way I was getting the date and time, I changed the date time property to use System